pkg/utils/apiobject/apiobject.go

```diff
@@ -27,3 +29,9 @@ func PodNamespacedNames(pods []*v1.Pod) []types.NamespacedName {
 	}
 	return namespacedNames
 }
+
+func MustHash(object interface{}) uint64 {
```

Comment:
This package is intended for `apiObject` helpers like `runtime.Object` `client.Object`, etc. I've been trying to deprecate it. I think `PodNamespacedNames` can probably go live in `pod` or w/e we end up naming it. Maybe we can make something like `must.Hash(obj)` or `hash.OrDie(obj)`